backend/app/api/ws/chat_namespace.py

```
@@ -1482,6 +1483,217 @@ async def on_skill_response(self, sid: str, data: dict) -> dict:
+                logger.error(
+                    f"[WS] interactive:response error: User not found id={user_id}"
+                )
+                return {"error": "User not found"}
+
+            # Get task
+            task = (
+                db.query(TaskResource)
+                .filter(
+                    TaskResource.id == task_id,
+                    TaskResource.kind == "Task",
+                    TaskResource.is_active == True,
```

Ruff (0.14.13)

Avoid equality comparisons to `True`; use `TaskResource.is_active:` for truth checks

Replace with `TaskResource.is_active`

(E712)